### PR TITLE
Add toString method to Redshift AWS API tasks

### PR DESCRIPTION
**Pull request by kpiotrowski**
Add `toString` method with information about AWS API call made by `RedshiftClusterNodesTask` and `RedshiftClusterUsageMetricsTask`

image
image

### CODE REVIEW

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java [hunk 1]
```diff
@@ -58,4 +58,16 @@ protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, Handle hand
             .collect(toList()));
     return null;
   }
+
+  private String getRequestDescription() {
+    return "AmazonRedshift.describeClusters";
+  }
+
+  @Override
+  public String toString() {
```
Comment (dawidxc) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java:67]: Please adapt this class to the new `describeSourceData` method (after my recent merge of #344).

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java [hunk 2]
```diff
@@ -58,4 +58,16 @@ protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, Handle hand
             .collect(toList()));
     return null;
   }
+
+  private String getRequestDescription() {
```
Comment (dawidxc) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java:62]: Consider renaming to `toCallDescription` method for consistency with:

https://github.com/google/dwh-migration-tools/blob/d77780f4825a2f3130d4e8942c5a4c1ded8b82c6/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/bigquery/AbstractBigQueryConnector.java#L174-L179

or:
https://github.com/google/dwh-migration-tools/blob/d77780f4825a2f3130d4e8942c5a4c1ded8b82c6/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/AbstractHiveConnector.java#L242-L247

**Review by dawidxc — APPROVED** [review]

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java (hunk 2 above)
Comment (kpiotrowski) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java]: 👍

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java (hunk 1 above)
Comment (kpiotrowski) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java]: 👍

**Pull request merged**